opentelemetry-otlp/tests/integration_test/expected/metrics.json

```diff
@@ -6,114 +6,128 @@
           {
```

Comment:
This file was already here, but was seemingly unused, so i've repurposed it to match the style of the other tests